src/db/seeds.sql

```diff
@@ -0,0 +1,40 @@
+use company_db;
```

Comment:
This script works perfectly however would be even nicer to write a `seed` script so the user can just run `npm run seed` and have this done "automatically"